src/tools/dart-devtools.md

```diff
@@ -1,16 +1,73 @@
 ---
 title: Dart DevTools
-description: A suite of performance tools for Dart and Flutter.
+description: A suite of debugging and performance tools.
 toc: false
 ---
 
-Dart DevTools is a suite of performance tools for Dart and Flutter.
-In addition to Flutter-specific features,
-DevTools supports source-level debugging of Dart apps,
-as well as viewing general log and diagnostics information about a running app.
-No matter whether you use an IDE or the command line,
-DevTools works with your existing development workflow.
+Dart DevTools is a suite of debugging and performance tools for Dart and
+Flutter. In addition to [Flutter-specific features][Flutter DevTools documentation.],
+DevTools supports source-level debugging of Dart apps, as well as viewing
+general log and diagnostics information about a running app.
 
-For more information, see the [Dart DevTools documentation.][]
+## Using DevTools with a command-line app
 
-[Dart DevTools documentation.]: https://flutter.github.io/devtools/
+### Install DevTools
+
+If you have `pub` on your path, you can run:
+
+```console
```

Comment:
```terminal
    ```

Tends to give better results than `console` in Jekyll .md files (as opposed to .md files rendered by GitHub, like READMEs).

Reply:
Ah, good tip, that does look slightly better (it formats the `$` prefix with a different color than the succeeding command)